### PR TITLE
added more supported games

### PR DESCRIPTION
**Pull request by stamepicmorg**
# wtf :interrobang:

We are ([STAM](https://github.com/stamepicmorg) and [AurZum](https://github.com/Aleks-Z)) collected list of supported games from 
* [metamod Original](https://sourceforge.net/projects/metamod/files/Metamod%20Sourcecode/1.20/) (look at file "games.h"), 
* [metamod-p](https://github.com/jkivilin/metamod-p/blob/master/metamod/games.h) (old repo at [SF](https://sourceforge.net/p/metamod-p/code/ci/master/tree/metamod/games.h)) ([online list](http://metamod.org/supportedmods.html)),
* [metamod-am\metamod-hl1](https://github.com/alliedmodders/metamod-hl1/blob/master/metamod/games.h) by [alliedmodders](https://github.com/alliedmodders/)
* [metamod-p-cmake](https://github.com/SamVanheer/Metamod-P-CMake/blob/master/metamod/games.h) (just opmtimazed c-make fork).  

and we are merged it to one big cleanuped and actualized list. Why not?

Now we have checking mods to compablity with [ReHLDS](https://github.com/dreamstalker/rehlds)  and [Metamod-r](https://github.com/theAsmodai/metamod-r):
* we are cleaned list of available games and deleted old won-games. 'cause rehlds works only on steam-version -> metamod-r too, against original metamod. 

compablity tests table:
https://github.com/EpicMorgGames/metamod-r-game-tests

### CODE REVIEW

**Comment by In-line** [comment]
At least add here a check for `knownfn` https://github.com/theAsmodai/metamod-r/blob/e32e4f6d6ec660bed47a452e09706fe7dd786eca/metamod/src/game_support.cpp#L144

**Comment by In-line** [comment]
And yes, it is actually `rehlds` or `ReHLDS`, but not `reHLDS` see from original @dreamstalker post https://forums.alliedmods.net/showthread.php?t=264661 :shipit:

File: metamod/src/game_support.cpp [hunk 1]
```diff
@@ -10,21 +10,97 @@ const game_modinfo_t g_known_games[] = {
...
+	{ "valve",              "hl.so",                    "hl.dll",                    "Half-Life"                                        },
+	{ "vs",                 "vs.so",                    "mp.dll",                    "VampireSlayer"                                    },
+	{ "wantedhl",           "hl.so",                    "wanted.dll",                "Wanted!"                                          },
+	{ "wizardwars",         "wizardwars_i486.so",       "wizardwars.dll",            "Wizard Wars (Steam)"                              },
+	{ "wizardwars_beta",    "wizardwars_i486.so",       "wizardwars.dll",            "Wizard Wars Beta (Steam)"                         },
+	{ "wizwars",            "mp.so",                    "hl.dll",                    "Wizard Wars (Old)"                                },
+	{ "wormshl",            "wormshl_i586.so",          "wormshl.dll",               "WormsHL (Old)"                                    },
+	{ "wormshl",            "wormshl_i686.so",          "wormshl.dll",               "WormsHL (Steam)"                                  },
```
Comment (In-line) [inline metamod/src/game_support.cpp]: There is already a code automatically lookup for all postfixes for linux versions.
See https://github.com/theAsmodai/metamod-r/blob/2c41e0ae1d629220794e30ba2c89c4a0076bdb95/metamod/src/utils.cpp#L3

Comment (stamepicmorg) [reply metamod/src/game_support.cpp]: ok. and how it will be palced?  "wormshl_i586.so" -> "wormshl"?

Comment (In-line) [reply metamod/src/game_support.cpp]: @stamepicmorg `wormshl.so`

Comment (stamepicmorg) [reply metamod/src/game_support.cpp]: i am not shure but updated it at commit https://github.com/theAsmodai/metamod-r/pull/25/commits/7a0a9953b3ed579b82ac4fcfb9ca86ee96cb4d78

check plz?

File: metamod/src/game_support.cpp [hunk 2]
```diff
@@ -69,15 +69,14 @@ const game_modinfo_t g_known_games[] = {
...
 	{ "scihunt",            "shunt.so",                 "shunt.dll",                 "Scientist Hunt"                                   },
 	{ "ship",               "ship.so",                  "ship.dll",                  "The Ship"                                         },
 	{ "si",                 "si.so",                    "si.dll",                    "Science & Industry"                               },
 	{ "snow",               "snow.so",                  "snow.dll",                  "Snow-War"                                         },
 	{ "stargatetc",         "hl.so",                    "hl.dll",                    "StargateTC (Old, 1.x)"                            },
-	{ "stargatetc",         "stc",                      "hl.dll",                    "StargateTC (Updated, 2.x)"                        }, // _i386.so
+	{ "stargatetc",         "stc.so",                   "hl.dll",                    "StargateTC (Updated, 2.x)"                        }, // _i386.so
 	{ "stargatetc",         "stc_i386_opt.so",          "hl.dll",                    "StargateTC (Updated, 2.x)"                        }, // ...
```
Comment (stamepicmorg) [inline metamod/src/game_support.cpp]: but i still not shure about this file. 

"stc_i386_opt.so" -> "stc_opt.so" ??

**Comment by stamepicmorg** [comment]
Anyone?

@In-line ? @theAsmodai?

Check PR plz ❤️

**Comment by stamepicmorg** [comment]
@In-line ? @theAsmodai? @s1lentq?

Check PR plz ❤️

File: metamod/src/game_support.cpp (hunk 2 above)
Comment (In-line) [reply metamod/src/game_support.cpp]: no, it should remain `stc_i386_opt.so`

**Pull request merged**